src/foundry/client/core/issues.d.mts

```diff
@@ -87,9 +87,17 @@ declare global {
      * Add a Document to the count of module-provided sub-types.
      * @param documentName      - The Document name.
      * @param subType           - The Document's sub-type.
-     * @param options.decrement - Decrement the count rather than incrementing it. default: `false`
      */
-    #countDocumentSubType(documentName: DocumentType, subType: string, options?: { decrement?: boolean }): void;
+    #countDocumentSubType(
+      documentName: DocumentType,
+      subType: string,
+      options?: {
+        /** Decrement the counter rather than incrementing it.
```

Comment:
While you're fixing the `protected` bit, can you also bump this down a line so it's
```ts
/**
 * Decrement...
 ```